test/ch/imedias/rsccfx/model/RsccTest.java

```diff
@@ -149,4 +149,41 @@ public void testValidateKey() {
       assertTrue(model.validateKey(validKey));
     }
   }
+
+  /**
+   * Test for {@link Rscc#startVncServer()}.
+   */
+  @Test
+  public void testStartVncServer() {
```

Comment:
those scenarios are not yet implemented, we can do tests whenever we know what we are going to have. This is covered by codeCoverage